pkg/pillar/dpcmanager/dpc.go

```diff
@@ -320,7 +326,7 @@ func (m *DpcManager) compressDPCL() {
 				i, dpc)
 		} else {
 			// Retain the lastresort if enabled. Delete everything else.
-			if dpc.Key == LastResortKey && m.enableLastResort {
+			if (dpc.Key == LastResortKey && m.enableLastResort) || (dpc.Key == ManualDPCKey) {
 				m.Log.Tracef("compressDPCL: Retaining last resort. i = %d, dpc: %+v",
```

Comment:
trace text doesn't capture the manual case. Maybe saying "manual or last resort"?